### DISCUSSION TITLE
How to run build script

### DISCUSSION THREAD

**Discussion by siyavash4812** [oblador/loki#340] [Q&A]
I am trying to add odiff to this project but I cannot build the repo to make sure it is working as expected. I want to install the package using my forked version but I keep getting the full source code rather than the build files.

Does anyone know how I can get the build/dist files?
Thank you

**Comment by techeverri** [comment]
There is no build step in Loki's release process, the full source is published (unbundled, un-transpiled/un-copiled) as a package(s)
Example: https://unpkg.com/browse/loki@0.28.1/

**Comment by techeverri** [comment]
If you want to test that the changes in your fork work as expected, follow the instructions here 👇 
https://github.com/oblador/loki/blob/master/packages/loki/README.md#development
